doc/using_pytorch.rst

```diff
@@ -566,94 +566,120 @@ The function should return a byte array of data serialized to content_type.
 The default implementation expects ``prediction`` to be a torch.Tensor and can serialize the result to JSON, CSV, or NPY.
 It accepts response content types of "application/json", "text/csv", and "application/x-npy".
 
-Working with Existing Model Data and Training Jobs
-==================================================
 
-Attach to existing training jobs
---------------------------------
+Bring your own model
+====================
 
-You can attach a PyTorch Estimator to an existing training job using the
-``attach`` method.
+You can deploy a PyTorch model that you trained outside of SageMaker by using the ``PyTorchModel`` class.
+Typically, you save a PyTorch model as a file with extension ``.pt`` or ``.pth``.
+To do this, you need to:
+
+* Write an inference script.
+* Package the model artifacts into a ``tar.gz`` file.
+* Upload the ``tar.gz`` file to an S3 bucket.
```

Comment:
GH isn't letting me reply to https://github.com/aws/sagemaker-python-sdk/pull/1457#discussion_r422428119 directly, so starting a new comment thread here.

the only advantage I can think of for doing the packing oneself is better control over the S3 location and how the file is packed. however, I'm pretty sure everything is covered in through the Python SDK with specifying an S3 location, etc.

Reply:
So is this the `_upload_code` function in `FrameworkModel`? If so, does that mean that the user supplies the optional `code_location` param to the constructor to specify a bucket, or use the default session bucket?

And then the constructor uploads to that S3 location (either default session or `code_location)?

Reply:
yes, `PyTorchModel` takes care of repacking the model and uploading it, and the location can be defined by `code_location` if the user doesn't want to use the default session bucket.

Reply:
@eslesar-aws do you still lean toward keeping those two steps in or do you think it'd be better to remove them since `PyTorchModel` can take care of repacking the model and uploading it?